master/buildbot/test/unit/test_interpolate_secrets.py

```diff
@@ -74,8 +74,8 @@ def setUp(self):
         self.master = fakemaster.make_master(self)
         fakeStorageService = FakeSecretStorage()
         password = "bar"
-        secretdict = {"foo": password, "other": password + 'random'}
-        fakeStorageService.reconfigService(secretdict=secretdict)
+        fakeStorageService.reconfigService(
+            secretdict=dict(foo=password, other=password + "random", empty=""))
```

Comment:
Let's prefer regular dicts here.

Reply:
I assume you mean literal since they're both regular, but pushing the change :+1:

Reply:
You're right, I meant literal.